docs/website/contents/for-developers/Glossary.md

```diff
@@ -94,6 +94,29 @@ Chain sync is a mini protocol to exchange chains of headers.
 
   - The server can *claim to have no additional headers* when asked for updates. This is signaled explicitly via `MsgAwaitReply` or implicitly by the server's selection tip point that is sent with every message (though the latter is currently not used on the client for any logic).
 
+## :ChainSync Jumping
+
+In the context of [Genesis](#genesis-consensus-protocol-paper-version), a node should execute [ChainSync](#chainsync) with all of its peers, thus often re-downloading and re-validating the same header from multiple peers. This would unnecessarily increase the network load on honest peers, which by definition will serve the same headers on the historical part of the chain.
+
+To mitigate this load, we use ChainSync Jumping, an extended version of ChainSync in which a node only download headers from *one* of its peers, and periodically asks all the other peers if they agree with it. If there is a disagreement, ChainSync starts with another peer until one of them gets disconnected.
+
+### :Dynamo
+
+The dynamo is the only peer from which we get all headers, and run the normal ChainSync protocol. Every once in a while, a node sends a message to all of its other peers to check if the tip of the dynamo's chain is also on their chain.
+
+### :Jumper
+
+The jumpers are all peers of a node which agree on the tip of its dynamo. Once they confirm they have that tip on their chain, the node assumes that they have the exact same chain as the dynamo, up to the current tip.
```

Comment:
I think this is only true of the “happy” jumpers. The other jumpers do not agree with the tip of the dynamo, they just haven't been promoted to objector yet.